converter/src/main/java/gov/cms/qpp/conversion/decode/ClinicalDocumentDecoder.java

```diff
@@ -51,7 +51,9 @@ protected DecodeResult decode(Element element, Node thisNode) {
 		setProgramNameOnNode(element, thisNode);
 		setEntityIdOnNode(element, thisNode);
 		setPracticeSiteAddress(element, thisNode);
-		setNationalProviderIdOnNode(element, thisNode);
+		if (ENTITY_INDIVIDUAL.equals(thisNode.getValue(ENTITY_TYPE))) {
```

Comment:
Does this work correctly for CPC+?

Reply:
tests are passing and CPC+ submissions are regarded as individual.

```
	private String[] getProgramNameEntityPair(String name) {
		String[] pairs;
		if (MIPS_INDIVIDUAL.equalsIgnoreCase(name)) {
			pairs = new String[] {MIPS_PROGRAM_NAME, ENTITY_INDIVIDUAL};
		} else if (MIPS_GROUP.equalsIgnoreCase(name)) {
			pairs = new String[] {MIPS_PROGRAM_NAME, ENTITY_GROUP};
		} else if (CPCPLUS.equalsIgnoreCase(name)) {
			pairs = new String[] {CPCPLUS_PROGRAM_NAME, ENTITY_INDIVIDUAL};
		} else {
			pairs = new String[] {name.toLowerCase(Locale.ENGLISH), ENTITY_INDIVIDUAL};
		}
		return pairs;
	}
```

Reply:
Ah right.  All good.